website/sidebars.js

```diff
@@ -466,10 +467,10 @@ module.exports = {
       label: "dbt Artifacts",
       items: [
         "reference/artifacts/dbt-artifacts",
-        "reference/artifacts/manifest",
-        "reference/artifacts/run-results",
-        "reference/artifacts/catalog",
-        "reference/artifacts/sources",
+        "reference/artifacts/manifest-json",
```

Comment:
For anyone else that has the same instinct: these files don't exist in `current`, so I don't feel the need to add the changed paths to the `_redirects` file :)